app/src/main/java/org/dosomething/letsdothis/ui/fragments/CategoryFragment.java

```diff
@@ -133,13 +143,34 @@ public void onCampaignExpanded(int position)
         recyclerView.smoothScrollToPosition(position);
     }
 
+    @Override
+    public void onReportBackClicked(int reportBackId)
+    {
+        startActivity(ReportBackDetailsActivity.getLaunchIntent(getActivity(), reportBackId));
+    }
+
+    @Override
+    public void onScrolledToBottom()
+    {
+        if(currentPage < totalPages)
+        {
+            Toast.makeText(getActivity(), "get more data", Toast.LENGTH_SHORT).show();
+            ReportBackListTask task = new ReportBackListTask(position,
+                                                             StringUtils.join(sampleIdsSubset, ","),
```

Comment:
task only used in one place, skip the local variable? if its for readibility then make the string a variable?